wnprc_billing/resources/views/updateProgramIncomeAccount.html

```diff
@@ -0,0 +1,130 @@
+                    if (success) {
+                        var jsonResponse = JSON.parse(xhr.responseText);
+                        var currentCreditToAccount = jsonResponse.creditToAccount;
+                        var currentJETCSVSetting = jsonResponse.jetCSVSetting;
+                        document.getElementById("ctaCell1").innerHTML = currentCreditToAccount;
+                        document.getElementById("jetcsvCell1").innerHTML = currentJETCSVSetting;
+                    }
+                    else {
+                        alert('Couldn\'t get program income account.');
+                    }
+                }
+            });
+        }
+
+        //This function verifies the user has the EHRFinanceAdminPermission, then sets the new
+        // values for 'CreditToAccount' and 'JetCsvSetting'.
+        function setValues(propertyName, newValue) {
+            if ((propertyName !== "") && (newValue !== "")) {
+                //Verifies the current user has EHRFinanceAdminPermission.
+                LABKEY.Ajax.request({
+                    url: LABKEY.ActionURL.buildURL('wnprc_billing', 'checkFinanceAdminStatus', null, {}),
+                    callback: function (config, success, xhr) {
+                        if (success) {
+                            //Sets the new module property values.
+                            let parentContainer = LABKEY.Security.currentContainer.parentId;
+                            let newPropertyValue = newValue;
+                            let propertyToUpdate = propertyName;
+                            let moduleToUpdate = "wnprc_billing"
+
+                            //Creates an object with the new module property values.
+                            var newProperties = [];
+                            newProperties.push({
+                                container: parentContainer,
+                                value: newPropertyValue,
+                                propName: propertyToUpdate,
+                                moduleName: moduleToUpdate
+                            });
+
+                            //Updates properties.
+                            LABKEY.Ajax.request({
+                                url: LABKEY.ActionURL.buildURL('core', 'saveModuleProperties', null),
+                                method: "POST",
+                                jsonData: {
+                                    properties: newProperties
+                                },
+                                headers: {
+                                    'Content-Type' : 'application/json'
+                                },
+                                callback: function (config, success, xhr) {
+                                    if (success) {
+                                        getValues();
+                                        document.getElementById("field1").value = "";
+                                        document.getElementById("field2").value = "";
```

Comment:
I noticed that this is clearing both form values. Is it possible to clear only the value that gets updated?

Reply:
Yep!  I already pass 'propertyName' into setValues() so I'll just write an if/else statement based on which one is being targeted.